src/controllers/auth.js

```diff
@@ -182,6 +185,47 @@ class AuthController {
             res.status(500).json({ errors: err });
         }
     }
+
+    static async confirmNewPassword(req, res) {
+        const { password, resetPasswordToken } = req.body;
+
+        try {
+            const tokenData = jwt.verify(resetPasswordToken, process.env.TOKEN_SECRET);
```

Comment:
You check you get the value of "process.env.TOKEN_SECRET"? It can be undefined if not use - [dotenv](https://www.npmjs.com/package/dotenv) . Good to have some king of config.js file where you get all variables from .env
[config file](https://github.com/max-logvyniuk/true_falser_server/blob/master/api/server/src/config/config.js)

Reply:
added config file